sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Generated/TopologyOperationsExtensions.cs

```diff
@@ -56,9 +56,13 @@ public static async Task<IPage<TopologyResource>> ListAsync(this ITopologyOperat
             /// <param name='operations'>
             /// The operations group for this extension method.
             /// </param>
-            public static IPage<TopologyResource> ListByHomeRegion(this ITopologyOperations operations)
+            /// <param name='ascLocation'>
+            /// The location where ASC stores the data of the subscription. can be
+            /// retrieved from Get locations
+            /// </param>
+            public static IPage<TopologyResource> ListByHomeRegion(this ITopologyOperations operations, string ascLocation)
```

Comment:
the ci reported the code is not same as auto-gened, so please don't change any code under the generated folder